dp/cloud/python/magma/configuration_controller/crl_validator/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""
```

Comment:
[dp/cloud/python/magma/__init__.py](https://github.com/magma/magma/pull/13359/files#diff-b6ec112d156f2e00319f5f0cdb1c6aacc529d290b25d023e9b6d16d64504d4d6) is missing header

Reply:
Removed the file, as it's unnecessary and was pushed by mistake.